azurerm/resource_arm_monitor_action_group_test.go

```diff
@@ -254,8 +537,51 @@ func TestAccAzureRMMonitorActionGroup_singleReceiverUpdate(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "enabled", "true"),
 					resource.TestCheckResourceAttr(resourceName, "email_receiver.#", "1"),
 					resource.TestCheckResourceAttr(resourceName, "email_receiver.0.email_address", "admin@contoso.com"),
+					resource.TestCheckResourceAttr(resourceName, "itsm_receiver.#", "0"),
+					resource.TestCheckResourceAttr(resourceName, "azure_app_push_receiver.#", "0"),
 					resource.TestCheckResourceAttr(resourceName, "sms_receiver.#", "0"),
 					resource.TestCheckResourceAttr(resourceName, "webhook_receiver.#", "0"),
+					resource.TestCheckResourceAttr(resourceName, "automation_runbook_receiver.#", "0"),
+					resource.TestCheckResourceAttr(resourceName, "voice_receiver.#", "0"),
+					resource.TestCheckResourceAttr(resourceName, "logic_app_receiver.#", "0"),
```

Comment:
This check failed:`Check failed: Check 11/13 error: azurerm_monitor_action_group.test: Attribute 'logic_app_receiver.resource_id' not found`

Reply:
I still need to look into this one.